### PR TITLE
config(tidb): rename needs cherry pick labels

### PR DESCRIPTION
**Pull request by wuhuizuo**
from `needs-cherry-pick-x.y` to `needs-cherry-pick-release-x.y`.

### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

### CODE REVIEW

File: prow/config/labels.yaml [hunk 1]
```diff
@@ -4235,49 +4235,49 @@ repos:
...
         description: "Type: Need cherry pick to release-5.3"
         target: both
-      - name: needs-cherry-pick-5.4
+      - name: needs-cherry-pick-release-5.4
         color: "000000"
         description: "Type: Need cherry pick to release-5.4"
         target: both
-      - name: needs-cherry-pick-6.0
+      - name: needs-cherry-pick-release-6.0
         color: "ededed"
```
Comment (Mini256) [inline prow/config/labels.yaml:4268]: use the same  color

**Review by purelind — APPROVED** [review]
lgtm

**Review by Mini256 — APPROVED** [review]
rest LGTM

File: prow/config/labels.yaml (hunk 1 above)
Comment (wuhuizuo) [reply prow/config/labels.yaml:4268]: some days before, some PingCAPer ask to highlight the latest release label.

**Pull request merged**